requirements.txt

```diff
@@ -1 +1 @@
-neuraxle>=0.2.1
```

Comment:
I would prefer to specify `neuraxle>=0.3.0` (and without Camel Case) to force Neuraxle to be at least version 0.3.0. Although, it's fine for now to not specify a version / that's not too bad.